docs/modules/control/move_to_a_pose_control/move_to_a_pose_control.rst

```diff
@@ -5,6 +5,12 @@ This is a simulation of moving to a pose control
 
 .. image:: https://github.com/AtsushiSakai/PythonRoboticsGifs/raw/master/PathTracking/move_to_pose/animation.gif
 
+For a more advanced version, see move_to_pose_robot_class.py. The program 
```

Comment:
If possible, could you please add a mathematical note of this controller?

Reply:
Yes. I was working on it yet. One thing that I notice is that the Kp_beta must be negative which does not make sense.

Reply:
If you want, you can fix #600 in this PR.

Reply:
I fixed reviewed the issue #600 and committed it in this PR. It was kinda difficult to catch it. I explain the logic here: Both 'Kp_alpha * alpha' and  'Kp_beta * beta' are significant values even at the vicinity of the target. When rho = ~0 and the heading is not aligned, the combined terms (Kp_alpha * alpha - Kp_beta * beta') rotates the robot toward the target direction. The reason that we need a negative sign for Kp_beta is that when we are far from the target we need an offset in the opposite direction so can help the robot to smoothly approach the target direction otherwise if Kp_beta = 0 the robot directly goes to the target (x,y) position. Another way to think about the negative sign of the Kp_beta is the way we take turns bicycle. For example, we initially lean left when we want to take a right turn. Indeed for the bicycle turning, the main reason is due to the dynamic motion. It was just a simple analogy to explain -Kp_beta easier...